tests/Operation/AggregateFunctionalTest.php

```diff
@@ -110,20 +110,23 @@ public function testTypeMapOption(array $typeMap = null, array $expectedDocument
         $this->assertEquals($expectedDocuments, $results);
     }
 
-    public function testTypeMapInline()
+    /**
+     * @dataProvider provideTypeMapOptionsAndExpectedDocuments
+     */
+    public function testTypeMapOptionWithoutCursor(array $typeMap = null, array $expectedDocuments)
```

Comment:
`$expectedDocuments` isn't being used for anything. Can you add `$this->assertEquals($expectedDocuments, $results);` (as `testTypeMapOption()` does) down below?